tests/integration/request_integration.cpp

```diff
@@ -403,6 +403,26 @@ TEST(request, should_send_and_receive_empty_optional) {
     io.run();
 }
 
+TEST(request, should_send_and_receive_interval) {
+    using namespace ozo::literals;
+
+    const auto interval = ozo::pg::interval(239278272013014LL);
+
+    ozo::io_context io;
+    const ozo::connection_info<> conn_info(OZO_PG_TEST_CONNINFO);
+    const auto timeout = ozo::time_traits::duration::max();
```

Comment:
No needs to make a timeout equal to `ozo::time_traits::duration::max()`, you can use the timeout-free version of the function.